rag_experiment_accelerator/config/config.py

```diff
@@ -102,7 +102,8 @@ def __init__(
             else ChunkingStrategy.BASIC
         )
         self.AZURE_DOCUMENT_INTELLIGENCE_MODEL = config_json.get(
-            "azure_document_intelligence_model", "prebuilt-read")
+            "azure_document_intelligence_model", "prebuilt-read"
```

Comment:
I'm surprised that these are here, looks like the linters are fighting again ⚔️

Reply:
The problem is, linter seems happier with this version of the file, so I might have to merge that